### PR TITLE
Add apps/navigation bar expanded title properties to the NAVIGATION_BAR configuration

### PR DESCRIPTION
**Pull request by jmpg93**
Changes to be able to communicate the navigation bar large title status using the bridge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,6 +248,7 @@ updateNavigationBar = ({
 ```
 
 -   `title`: updates NavigationBar title
+-   `expandedTitle`: updates NavigationBar expandedTitle
```
Comment (atabel) [inline README.md]: document here that empty string means no expanded title

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # CHANGELOG
 
-## v.2.11.0 - 2019-03-02
+## v.2.11.1 - 2020-03-10
```
Comment (pladaria) [inline CHANGELOG.md]: this should be a minor version upgrade (2.12.0)

File: README.md (hunk 1 above)
Comment (pladaria) [reply README.md]: add some information about the minimum required app version to use this property

File: src/__tests__/utils-test.ts [hunk 3]
```diff
@@ -119,6 +120,7 @@ test('update navigation bar, without options', cb => {
 test('update navigation bar, with options', cb => {
     const options = {
         title: ANY_STRING,
+        expandedTitle: ANY_STRING,
```
Comment (pladaria) [inline src/__tests__/utils-test.ts]: I'd use different strings, imagine some weird bug in the implementation where the expandedTitle is sent as title, this test wouldn't catch it

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 # CHANGELOG
```
Comment (pladaria) [inline CHANGELOG.md]: I'm missing changes in `src/webview-bridge-cjs.js.flow`

That file contains flow type definitions (flow is the typesystem we use in webapp)

File: README.md (hunk 1 above)
Comment (jmpg93) [reply README.md]: Sure! Updated.

File: CHANGELOG.md (hunk 4 above)
Comment (jmpg93) [reply CHANGELOG.md]: You are right. Thanks for the clarification.

File: src/__tests__/utils-test.ts (hunk 3 above)
Comment (jmpg93) [reply src/__tests__/utils-test.ts]: Sure.

**Pull request merged**